background/transport.js

```diff
@@ -236,16 +237,34 @@ export default class Transport {
       this.refreshAccountsCache();
 
       // fetch version
-      secureTransport.execute('web3_clientVersion')
-        .then(version => {
+      Promise.all([
+        secureTransport.execute('web3_clientVersion').then(version => {
           State.version = version;
+          return version;
+        }),
+        this.getChainName()
+      ])
+        .then(([version, chain]) => {
+          Config.set({
+            lastVersion: version,
+            lastChain: chain
+          });
+
+          analytics.ifEnabled(() => {
+            analytics.set('dimension1', version);
+            analytics.set('dimension2', chain);
```

Comment:
Constants are a good idea, will change that and give the constants more meaningful names with a comment why the values are so weird. Thanks!